meter/goodwe-wifi.go

```diff
@@ -0,0 +1,159 @@
+package meter
+
+import (
+	"encoding/binary"
+	"net"
+	"time"
+
+	"github.com/evcc-io/evcc/api"
+	"github.com/evcc-io/evcc/util"
+)
+
+type GoodweServer struct {
```

Comment:
Die Typen haben nichts im public scope zu suchen. Entweder private oder Du verschiebst das in einen goodwe Unterordner.

Reply:
hab sie jetzt umbenannt. Passts so? Sry bin in go nicht so gut bewandert.